python/F3DPythonBindings.cxx

```diff
@@ -223,6 +223,7 @@ PYBIND11_MODULE(pyf3d, module)
     .def("record_interaction", &f3d::interactor::recordInteraction, "Record an interaction file")
     .def("start", &f3d::interactor::start, "Start the interactor")
     .def("stop", &f3d::interactor::start, "Stop the interactor")
+    .def("trigger_command", &f3d::interactor::triggerCommand, "Trigger a command")
```

Comment:
@Meakk do you know If I can add callback bindings ?

Reply:
I think it should be possible. See https://pybind11.readthedocs.io/en/stable/advanced/cast/functional.html  
What happens if you just add `.def("add_callback", &f3d::interactor::addCallback, "Add a new command")`?